src/models/AccessRights.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * LINKURIOUS CONFIDENTIAL
+ * Copyright Linkurious SAS 2012 - 2019
+ *
+ * - Created on 2019-09-12.
+ */
+
+// TS2019-DONE
+
+import {IDataSourceParams} from './DataSource';
+
+export interface ISetAccessRightsParams extends IDataSourceParams {
```

Comment:
A comment with example values for each field would be welcome, otherwise it's difficult to guess what each field should be for a user.